backend/src/services/user/user-fns.ts

```diff
@@ -19,3 +21,54 @@ export const normalizeUsername = async (username: string, userDAL: Pick<TUserDAL
     }
   }
 };
+
+export const processFailedMfaAttempt = async (userId: string, userDAL: Pick<TUserDALFactory, "transaction">) => {
+  try {
+    const updatedUser = await userDAL.transaction(async (tx) => {
+      const PROGRESSIVE_DELAY_INTERVAL = 3;
+      const [user] = await tx(TableName.Users)
```

Comment:
Kindly use dal layer and not knex functions directly. To do increment, the update operation orm has support for it.

https://github.com/Infisical/infisical/pull/1837 but i think its still in review

Reply:
for now, I created a DAL method for this 
<img width="763" alt="image" src="https://github.com/Infisical/infisical/assets/65645666/bcfb297e-f39c-4590-97a2-214bf3b1625a">

Reply:
done @maidul98 !